src/Persistence/MassTransit.EntityFrameworkCoreIntegration/Configuration/IEntityFrameworkSagaRepositoryConfigurator.cs

```diff
@@ -42,6 +42,11 @@ void AddDbContext<TContext, TImplementation>(Action<IServiceProvider, DbContextO
         /// <typeparam name="TContext"></typeparam>
         void ExistingDbContext<TContext>()
             where TContext : DbContext;
+
+        /// <summary>
+        /// Configure whether to use transactions. By default, transactions are enabled.
+        /// </summary>
+        void ConfigureTransaction(bool isEnabled);
```

Comment:
This method should be removed, and replaced with the single method mentioned.

`EnableOptimisticConcurrency(bool useTransaction = true)`